File: Source/Libraries/LS.Events/PromptApi/GoalTaskResource.cs
namespace LS.Events.PromptApi;

public class GoalTaskResource
{
    public Guid Id { get; init; }
    public Guid GoalId { get; init; }
    public string Title { get; init; }
    public string Content { get; init; }
    public int EstimatedDuration { get; init; }
    public bool IsCompleted { get; set; }
    public DateTime Date { get; set; }
}

File: Source/Web/src/utils/api/prompt/models/GoalTaskResource.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { TimeSpan } from './TimeSpan';

export type GoalTaskResource = {
    id?: string;
    goalId?: string;
    content?: string | null;
    estimatedDuration?: TimeSpan;
    isCompleted?: boolean;
    date?: string;
};
